modules/scheduler/scheduler/src/main/java/org/motechproject/scheduler/service/MotechSchedulerService.java

```diff
@@ -45,15 +40,15 @@
     /**
      * Schedules the given schedulable job.
      *
-     * @param job  the job DTO
+     * @param job the SchedulableJob
      */
     void scheduleJob(SchedulableJob job);
 
     /**
-     * Schedules the given schedulable job. The Job ID by which the job will be referencing in the future should be provided
-     * in an Instance of MotechEvent in SchedulableJob (see MotechEvent.jobId)
+     * Schedules the given cron schedulable job. The Job ID by which the job will be referencing in the future should be provided
+     * in an Instance of MotechEvent in SchedulableJob (see motechEvent.getParameters().get("JobID"))
```

Comment:
the tag 'see' should indicate other method/variable. After changes in scheduler I think this see should be removed